### PR TITLE
Add evaluation scores to course user overview page

### LINKED ISSUE #2714: Add evaluation scores to course-scoped user profile
Right now, the evaluation scores of a user might be separated over multiple evaluations. Bringing them together on the course-scoped user profile might add value to see how a student has performed throughout a course, e.g. while giving personal feedback.

### PR DESCRIPTION
**Pull request by jorg-vr**
This pull request adds the results from evaluations to the course user overview page.

I reused the component from the series overview page that a student has access to.

I put it inside a details tag to reduce vertical space when not desired.
image
image
Each feedback links to its edit page, should the teacher desire to continue working on it.

This pr also contains a bugfix for a bug that appeared when updating the users of an existing evaluation.

Closes #2714

### CODE REVIEW

**Review by bmesuere — COMMENTED** [review]
👍 Is there a reason why we don't show the total score for each evaluation? I would even add that in the collapsed view.

**Comment by jorg-vr** [comment]
Not really, I reused another component and did not think about it. Will add it now

**Review by dodona-server — APPROVED** [review]

File: app/views/evaluations/_overview.html.erb
Comment (chvp) [inline app/views/evaluations/_overview.html.erb:1]: There's some weird indentation in this file.

File: app/views/evaluations/_overview.html.erb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                        </span>
+            <% end %>
+          <% else %>
+                      <span data-bs-toggle="tooltip" data-bs-placement="top" title="<%= t ".invisible_total" %>">
+                        -
+                      </span>
+          <% end %>
+        </strong>
+      </td>
+      <td class="actions align-top">
```
Comment (chvp) [inline app/views/evaluations/_overview.html.erb:69]: This `td` isn't closed.

**Review by chvp — APPROVED** [review]

**Review by niknetniko — APPROVED** [review]

**Pull request merged**